### PR TITLE
V0.14.1

### PR DESCRIPTION
**Pull request by jni**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -40,14 +41,18 @@ requirements:
     - dask-core >=0.15
     - toolz >=0.7.4
     - pywavelets >=0.4.0
-    - dask-core >=0.9.0
```
Comment (hmaarrfk) [inline recipe/meta.yaml]: Why are you requiring 0.15 on conda but 0.9 on pip?

Comment (jni) [reply recipe/meta.yaml]: a) none of these changes are mine. I had to fix some merge conflicts between your commits and master, and master was already on 0.15.
b) I suspect that the combination of Python 3.6 / dask-core only exists in certain versions, so it's probably not a huge point?
c) does dask-core < 0.15 even exist on conda? Actually, no, it doesn't, just checked.

Comment (hmaarrfk) [reply recipe/meta.yaml]: a) I wonder who made those changes.

Comment (jni) [reply recipe/meta.yaml]: The truth is out there. =P

**Comment by rth** [comment]
I'm wondering if this is still necessary?

From what I can see, 0.14.1 is available on conda forge (at least on Linux) and master appears to be re-rendered with conda-smithy 3.1.12 already. Maybe this can be closed then?

**Comment by hmaarrfk** [comment]
Yeah, this https://github.com/conda-forge/scikit-image-feedstock/pull/31 was merged a while back

**Comment by jni** [comment]
Yep, I lose track of conda-forge releases with all the bots now.

**Pull request closed without merging**